practice/spec/regex/regex_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe 'Regex pattern' do
+  before(:each) do
+    @book_path = File.join(
+        Dir.home, 'RubyTasks', 'practice', 'lib', 'regex', 'test_data', 'Story.txt'
+    )
+    @file_path = File.join(
+        Dir.home, 'RubyTasks', 'practice', 'lib', 'regex', 'test_data', 'alphanumeric.txt'
+    )
+    @NOT_EXISTING_STRING = 'Can not be in book'.freeze
+    @ONE_OCCURRENCE = 'Hello'.freeze
+    @MORE_TEN_OCCURRENCES = '(\s|\A)(come)([\s.,:]|\z)'.freeze
+    @ALPHA_NUMERIC = '(\s|^)[A-Za-z0-9]{20}(\s|$)'.freeze
```

Comment:
There is no need to use @ before Constants. Also, it's better to have them at the top.
```
describe 'Name' do
  Constant = something.
  ....
  Constant = something.
  before(:each)
```